frontend/packages/dev-console/src/components/custom-resource-list/custom-resource-list-types.ts

```diff
@@ -0,0 +1,20 @@
+import { FirehoseResult } from '@console/internal/components/utils';
+
+export enum ListFilterType {
+  Row = 'row',
+  Text = 'text',
+}
+
+export interface ListProps {
+  items: Promise<any>;
+  queryArg: string;
+  rowFilters: any;
+  resourceRow: React.FC;
```

Comment:
Could you add prop types here?
```suggestion
  resourceRow: React.ComponentType<...>;
```